core/src/main/java/org/elasticsearch/index/store/Store.java

```diff
@@ -211,17 +212,14 @@ private static SegmentInfos readSegmentsInfo(IndexCommit commit, Directory direc
     }
 
     /**
-     * Loads the local checkpoint and the maximum sequence number from the latest Lucene commit point and returns the triplet of local and
-     * global checkpoints, and maximum sequence number as an instance of {@link SeqNoStats}. The global checkpoint must be provided
-     * externally as it is not stored in the commit point.
+     * Loads the maximum sequence number and local checkpoint from the latest Lucene commit point.
      *
-     * @param globalCheckpoint the provided global checkpoint
-     * @return an instance of {@link SeqNoStats} populated with the local and global checkpoints, and the maximum sequence number
+     * @return a tuple populated with the maximum sequence number and the local checkpoint
      * @throws IOException if an I/O exception occurred reading the latest Lucene commit point from disk
      */
-    public SeqNoStats loadSeqNoStats(final long globalCheckpoint) throws IOException {
+    public Tuple<Long, Long> loadSeqNoStats() throws IOException {
```

Comment:
maybe call this load seqNoInfo? This way people won't be confused about SeqNoStats?

Reply:
ok